spk/influxdb/src/service-setup.sh

```diff
@@ -0,0 +1,21 @@
+INFLUXD_CONFIG_PATH=${SYNOPKG_PKGVAR}/config.yml
+INFLUXD=${SYNOPKG_PKGDEST}/bin/influxd
+
+export INFLUXD_CONFIG_PATH=${INFLUXD_CONFIG_PATH}
+
+SERVICE_COMMAND="${INFLUXD}"
+SVC_BACKGROUND=yes
+SVC_WRITE_PID=yes
+SVC_CWD="${SYNOPKG_PKGVAR}"
+
+service_postinst ()
+{
+    cat << EOF > "${INFLUXD_CONFIG_PATH}"
+http-bind-address: ":${SERVICE_PORT}"
+reporting-disabled: true
+bolt-path: "${SYNOPKG_PKGVAR}/.influxdbv2/influxd.bolt"
+engine-path: "${SYNOPKG_PKGVAR}/.influxdbv2/engine"
+sqlite-path: "${SYNOPKG_PKGVAR}/.influxdbv2/influxd.sqlite"
+
+EOF
+}
```

Comment:
This config is now static.
I highly recommend to add the config file to the package in the var folder.
This way it will automatically be installed.
Plus the config file in the package will not overwrite the config on package updates.